src/components/Topics/TopicList.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState, useEffect } from "react";
+
+const TopicList = () => {
+  const [topics, setTopics] = useState([]);
+
+  useEffect(() => {
+    fetch("https://back-end-nna.onrender.com/api/topics")
+      .then((res) => res.json())
+      .then((data) => setTopics(data.topics));
+  }, []);
+
+  return (
+    <ul>
+      {topics.map((topic) => (
+        <li key={topic.slug}>{topic.description}</li>
```

Comment:
This should be showing the `slug` instead of the `description` here as the slug is the name of the topic. You can optionally add the description as well, but the main info is the `slug` itself that we want to be displaying.